safe-room.lic

```diff
@@ -63,6 +65,33 @@ class SafeRoom
     score > @health_threshold
   end
 
+  def devour_healing(settings)
+    return unless settings.necromancer_healing['Devour']
+    return unless settings.thanatology['harvest_container']
+    
+    walk_to(settings.safe_room)
+    release_invisibility
+    devours = count_items_in_container('material', settings.thanatology['harvest_container'])
+    return unless devours > 0
```

Comment:
What do you think about `return if devours.zero?`?

Reply:
I feel like if I had done that to begin with, you'd be asking me why not `return unless devours > 0`.

Reply:
No, my default position is to use `if` over `unless`. It generally reads cleaner (obviously there are exceptions).

Reply:
Ok, gotcha.